Kitodo-DataManagement/src/main/java/org/kitodo/data/index/KitodoCustomLayoutStrategy.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.data.index;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.hibernate.search.backend.elasticsearch.index.layout.IndexLayoutStrategy;
+import org.kitodo.config.ConfigMain;
+
+/**
+ * Class usage is defined in hibernate.properties file as a value of hibernate.search.backend.layout.strategy key.
+ * Class is similar to the SimpleIndexLayoutStrategy implementation from Hibernate-Search but add an optional prefix
+ * in front of the index names.
+ */
+public class KitodoCustomLayoutStrategy implements IndexLayoutStrategy {
+
+    private String getSearchIndexPrefix() {
+        return ConfigMain.getParameter("searchindex.prefix", "");
+    }
+
+    @Override
+    public String createInitialElasticsearchIndexName(String hibernateSearchIndexName) {
+        return getSearchIndexPrefix() + hibernateSearchIndexName + "-000001";
```

Comment:
Adding `-000001` seems to be related to the "Index Lifecycle Management" module of ElasticSearch, in case the index is configured to "rollover" if it has reached a certain limit (e.g., the number of indexed documents, size in GB), similar to Linux log files can be configured to rollover every day or after a certain size is reached. 

Do we need that? I investigated this feature a bit and I don't think this works out-of-the-box at all. It has to be specifically configured in [ElasticSearch](https://www.elastic.co/docs/reference/elasticsearch/curator/rollover) and there does not seem to be full support for it in [HibernateSearch](https://hibernate.atlassian.net/browse/HSEARCH-3499) yet. Meaning, rewriting aliases to different indexes would need to be implemented manually.

If there is no reason for it (other than it was implemented like this in the example class provided by HibernateSearch), I think it would be better to remove `-000001` to not confuse some admins that might think Kitodo.Production uses this particular feature of ElasticSearch.

```suggestion
        return getSearchIndexPrefix() + hibernateSearchIndexName;
```

Reply:
I didn't realize this was the default naming strategy of hibernate search. I guess this is a good enough reason to keep it that way.